README.md

```diff
@@ -2,30 +2,22 @@
 
 Shared reusable react BCGov themed components.
 
-## Running Locally
+## Release packaging
 
-In order to use these shared react components locally, run:
+Run the following commands locally after a new component has been added:
 
-```bash
-yarn prepare
-```
+`bash yarn install`
```

Comment:
this will show up as `bash yarn install` which we dont want
should be:
```bash
yarn install
```
same below

Reply:
so use
```suggestion
```bash
yarn install
```
```